### DISCUSSION TITLE
G-Helper on Linux

### DISCUSSION THREAD

**Discussion by Konnos998** [seerge/g-helper#1723] [General]
Hello,

does G-Helper works on Linux (Pop!_OS to be specific), because I'm thinking of switching from Windows to Linux. And I've seen many videos telling that some of the functionality is missing in Linux (due to some apps don't being supported by Linux, like in Windows).
Does G-Helper has an installation for Linux too/can it run on Linux?

Thanks in advance.

**Comment by seerge** [comment]
For linux support you should check https://asus-linux.org/ 

You can't run G-Helper on linux obviously, as this is windows app ...

**Reply by Konnos998** [reply]
I see! I'll check the link you send me though.

Thank you

**Reply by KyrilosKamal** [reply]
> as
if it's available developer can make a version for linux, because asusctl amd supergfxctl are installed hardly really :|

**Reply by Algarues** [reply]
Is it known how to exactly replicate on linux the equivalent of Eco mode on Windows?
Even if through commands + configuration files?

In terms of battery saving, asusctl on linux can achieve only Quiet state. 
However, when compared to ECO on windows, on ROG Flow X13, residual heat from the GPU can still be felt on the chassis above the keyboard.

The heat can still be felt even if supergfxctl is set to "Integrated".

It is my impression that the dGPU is not completely switched off like on Windows.

Thanks for any help

**Comment by ouya-woelders** [comment]
Pls still make a Linux version, ghelper is way better than asusctl

**Reply by utajum** [reply]
On Ubuntu asusctl has never worked for me, here is an alternative :)

https://g-helper-linux.elevatech.xyz/

**Reply by MrXT21** [reply]
> On Ubuntu asusctl has never worked for me, here is an alternative :)
> 
> https://g-helper-linux.elevatech.xyz/

Thanks for posting that id never have found it. Legend. Do you know if that works on pop os?

**Comment by Algarues** [comment]
Is it possible to launch the app from the command line in Windows in debug mode?

I would like to understand the commands that are being used by the program to undervolt my CPU and try to replicate them using Ryzenadj in Linux.

Thank you

**Comment by keftie22** [comment]
For others looking here, someone has recently made their own sort of "port" of it here:

https://github.com/utajum/g-helper-linux

I haven't tested it myself yet but just from the screenshots it looks pretty close, probably covering most of the functionality you get from the Windows version. I haven't seen it get much attention on it yet but it might be good to get more eyes on it to help with testing different laptops and whatnot.

**Comment by Czarnykowal** [comment]
https://g-helper-linux.elevatech.xyz

works like a charm on cachyos

**Reply by Czarnykowal** [reply]
same stuff as a previous post

**Reply by ouya-woelders** [reply]
but does loose some functions such as switching gpu vrails on and off, it only disables the drivers and requires reboot sometimes.

**Reply by utajum** [reply]
@ouya-woelders not in the latest version, it depends on DE, model, what you have running in the background, but live dgpu disable works now + there is management for processes that hold the dgpu

**Reply by ouya-woelders** [reply]
@utajum interesting, previously when i tested it, it only disabled the driver but the gpu still had power, causing an extra 4w or so of power draw.
